gui/src/2D/controls/sliders/baseSlider.ts

```diff
@@ -301,7 +301,7 @@ export class BaseSlider extends Control {
             value = this._minimum + ((x - this._currentMeasure.left) / this._currentMeasure.width) * (this._maximum - this._minimum);
         }
 
-        const mult = (1 / this._step) | 0;
+        const mult = (1.0 / this._step);

```

Comment:
Note that it is a breaking change when `this._step < 1`.

Maybe you should do instead:
```
        const mult = this._step < 1 ? (1.0 / this._step) | 0 : (1.0 / this._step);
```